internal/mainflux/clients/status.go

```diff
@@ -1,6 +1,11 @@
 package clients
```

Comment:
You can move this to the root of `internal/mainflux` since in the current implementation clients and groups have the same status implementation